upi/openstack/common.yaml

```diff
@@ -24,3 +30,14 @@
       os_compute_server_group_name: "{{ infraID }}-worker"
       # Ignition files
       os_bootstrap_ignition: "{{ infraID }}-bootstrap-ignition.json"
+    when: sym.stat.exists
```

Comment:
What happens when `metadata.json` does not exist? Should the above task that check for existence error out when the file is not found?
Should we also do the same for `netid.json`?

Reply:
On the second thought - we can't fail this when there is no metadata.json, as it doesn't exists yet when the network playbook is run. I can check for netid.json file just in case, although it is created just a few steps before mentioned playbook is run.